CHANGELOG.md

```diff
@@ -6,6 +6,11 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Added
+- Tests to `insertCoupon` function.
+
+### Removed
+- `coupon` info in order to concentrate all the `orderForm` related data in `OrderManager`.
```

Comment:
I know there is an argument about removing these empty lines because it would help on git merge, avoiding conflicts, but prettier adds those spaces so the pattern has been add the empty lines (check the file below).